cypress/e2e/chi-custom-elements/alert.cy.js

```diff
@@ -138,4 +138,33 @@ describe('Alert', () => {
         .should('match', 'div.chi-alert.-toast');
     });
   });
+
+  describe('Alert Common', () => {
+    before(() => {
+      cy.visit('tests/custom-elements/alert-toast.html');
+    });
+
+    it('should change displayed title if attribute changes', () => {
+      cy.get('[data-cy="alert-toast-title-base-lg"]', { timeout: 5000 })
+        .scrollIntoView()
+        .as("alert")
+
+      cy.get('@alert')
+        .find('.chi-alert__title')
```

Comment:
Use constant class here: `ALERT_TITLE`

```suggestion
        .find(`.${ALERT_TITLE}`)
```